test/e2e/pipeline/diagnostic/diagnose.sh

```diff
@@ -0,0 +1,7 @@
+for i in $(seq 1 ${CLUSTER_NUMBER});
```

Comment:
Could be worth adding some information about the diagnose hook in the readme. Might also add some echo lines to describe each output.